sgl-pdlb/src/main.rs

```diff
@@ -0,0 +1,44 @@
+mod io_struct;
+mod server;
+mod strategy_lb;
+
+use server::{periodic_logging, startup};
+use tokio::signal;
+
+fn main() -> anyhow::Result<()> {
+    // FIXME: test code, move to test folder
+    let prefill_infos = (0..8)
+        .map(|i| (format!("123.123.123.123:{}", i), None))
+        .collect::<Vec<(String, Option<u16>)>>();
+
+    let decode_infos = (0..32)
+        .map(|i| format!("233.233.233.233:{}", i))
+        .collect::<Vec<String>>();
+
+    let lb_config = server::LBConfig {
+        host: "localhost".to_string(),
+        port: 8080,
+        policy: "random".to_string(),
+        prefill_infos,
+        decode_infos,
+        log_interval: 5,
+        timeout: 600,
+    };
+    let lb_state = server::LBState::new(lb_config.clone()).map_err(|e| anyhow::anyhow!(e))?;
+    let ret: anyhow::Result<()> = actix_web::rt::System::new().block_on(async move {
```

Comment:
nit: looks like there are some code dup b/t this and lib.rs, shall we extract to a function